data/drak/drak ships.txt

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2016 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+
+
+ship "Archon"
+	noun "entity"
+	sprite "ship/archon"
+		"frame rate" 0.87
+	thumbnail "thumbnail/archon"
+	"never disabled"
+	attributes
+		category "Heavy Warship"
+		"cost" 1000000000
+		"shields" 4000000
+		"hull" 1000000
+		"required crew" 1
+		"bunks" 1
+		"mass" 1000
+		"drag" 27
+		"heat dissipation" 40
+		"fuel capacity" 800
+		"cargo space" 0
+		"outfit space" 1300
+		"weapon capacity" 1125
+		"engine capacity" 200
+		"energy capacity" 10000
```

Comment:
Given that once it's angry it'll be firing twice as many weapons, and that the AMC already removes more than half the Archon's total energy cap (5610 out of 10000), I'd add another 0 to its energy capacity so that we don't have to worry about it lacking the energy at a particular instant.

Reply:
This comment had a duplicate because GitHub was acting weird and wouldn't let me add this to the review for a while. I deleted the duplicate (I think), just saying in case anyone got confused.

Reply:
Fixed. Thanks!